src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -4,75 +4,157 @@
 
 using System;
 using System.Device.Gpio;
-using System.Device.Spi;
 
 namespace Iot.Device.Mcp23xxx
 {
-    public class Mcp23xxx : IDisposable
+    // TODO: https://github.com/dotnet/iot/issues/125
```

Comment:
>/ [](start = 42, length = 1)

I added a comment on the issue but basically I'm not convinced that having these interfaces would be a great idea. The whole point of having a device binding is so that consumers will interact with that device class directly as opposed to having to interact with a GpioController. All of the interesting information that can be extracted from the device, should be wrapped by helper methods on that binding.

Reply:
Yup, that was leftover before change.  Should be updated now.

Reply:
Left comment on issue

Reply:
I'm going to go ahead and remove this TODO/comment since it hasn't been determined.  As mentioned, I think it would be a very nice feature, so let's keep pondering ideas.  We can always circle back and update the binding later.

On a side note, I'm working on ReadBit/WriteBit helpers and hope to update PR in the next few days.